src/ui/server/controllers/insurance/business/map-submitted-data/mocks/index.ts

```diff
@@ -11,7 +11,7 @@ export const mockBody = {
   [INPUT]: '8989898',
   __typename: 'CompaniesHouseResponse',
   ...mockCompany,
-  registeredOfficeAddress: {
+  address: {
```

Comment:
Shouldn't this be `registeredOfficeAddress` ?

Reply:
no, this is a temporary address which is processed and is passed to the api as `address` as `registeredOfficeAddress` is deleted as it contains nulls and unwanted fields like `__typename`